python/ray/ray_constants.py

```diff
@@ -278,3 +278,22 @@ def to_memory_units(memory_bytes, round_up):
 # The default gRPC max message size is 4 MiB, we use a larger number of 100 MiB
 # NOTE: This is equal to the C++ limit of (RAY_CONFIG::max_grpc_message_size)
 GRPC_CPP_MAX_MESSAGE_SIZE = 100 * 1024 * 1024
+
+# Internal kv namespaces
+KV_NAMESPACE_DASHBOARD = "dashboard"
+KV_NAMESPACE_SESSION = "session"
+KV_NAMESPACE_TRACING = "tracing"
+KV_NAMESPACE_PDB = "ray_pdb"
+
+# TODO: Set namespace for autoscaler
+# This will break backward compatibility between versions, where old version
+# autoscaler won't be able to scale up new version ray since the key will be
+# different with namespace.
+KV_NAMESPACE_AUTOSCALER = None
```

Comment:
Can we move this constant into the autoscaler package? 

Also is there a migration plan? I feel like we should probably make this read an env var (that isn't set by default).

Reply:
Hmm, I'm not sure about this. Any benefit of doing this?